server/athenian/api/models/precomputed/models.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime, timezone
+
+from sqlalchemy import CHAR, Column, func, Integer, LargeBinary, String, Text, TIMESTAMP
+from sqlalchemy.dialects.postgresql import HSTORE
+
+from athenian.api.models import create_base
+
+
+Base = create_base()
+
+
+class GitHubPullRequestTimes(Base):
+    """A group of repositories identified by an integer."""
+
+    __tablename__ = "github_pull_request_times"
+
+    pr_node_id = Column(CHAR(32), primary_key=True)
+    release_match = Column(Text(), primary_key=True)
+    repository_full_name = Column(String(64 + 1 + 100), nullable=False)
```

Comment:
Why not 165?

Reply:
This is to indicate the current GitHub limits:
- 64 chars for the user name
- 100 chars for the repo name